nemo/collections/asr/parts/jasper.py

```diff
@@ -36,6 +37,28 @@
 jasper_activations = {"hardtanh": nn.Hardtanh, "relu": nn.ReLU, "selu": nn.SELU, "swish": Swish}
 
 
+def tds_uniform_(tensor, mode='fan_in'):
+    """
```

Comment:
Is there any references for these initialization techniques to mention here?

Reply:
Yup I forgot to add it in a hurry.